Subject: libusb needs librt and libpthread

Add missing dependencies.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
# 20100603 rsc: The patch was closed upstream, and I don't
#          remember a use case any more :-( Let's keep it
#          here for the moment, until we have a user.
# 20100101 rsc: added as http://libusb.org/ticket/24

 libusb-1.0.pc.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: libusb-1.0.1/libusb-1.0.pc.in
===================================================================
--- libusb-1.0.1.orig/libusb-1.0.pc.in
+++ libusb-1.0.1/libusb-1.0.pc.in
@@ -6,6 +6,6 @@ includedir=@includedir@
 Name: libusb-1.0
 Description: C API for USB device access from Linux userspace
 Version: @VERSION@
-Libs: -L${libdir} -lusb-1.0
+Libs: -L${libdir} -lusb-1.0 -lrt -lpthread
 Cflags: -I${includedir}/libusb-1.0

